ocaml/testsuite/tests/typing-layouts-missing-cmi/function_arg.ml

```diff
@@ -10,8 +10,6 @@ module = "function_b.ml"
 script = "rm -f function_a.cmi"
 ***** expect
 *)
-(* CR layouts v2.9: all error messages below here are unreviewed *)
```

Comment:
It's also odd that the number in the name of the _representable_layout variable keeps ticking up in this file. Usually the "scope" of such an invented variable is just for the current error message -- we don't need for it to be unique across multiple definitions.

Reply:
filed ticket